CONTRIBUTING.rst

```diff
@@ -0,0 +1,75 @@
+Contributing to this repository
+===============================
+
+|:smile:| |:heart_eyes:| First off, thanks for taking the time to contribute! With your help, we’re making developers' lives easier. |:heart_eyes:| |:smile:|
+
+The following is a set of tips and guidelines that will help you improve the Aiven developer portal.
+
+What should I know before I contribute?
+---------------------------------------
+
+**What is devportal?**
+Devportal is Aiven’s developer portal. It’s goal is to provide a comprehensive set of help articles aimed at developers using Aiven’s products. 
+
+**How does devportal work?**
+It is Python-based, with content in ReStructuredText (rst) and rendered by Sphinx.
+
+**What is the content model?**
+Devportal contains the following content types:
+- *Concept* - Conceptual content helps people understand a feature or topic by providing a clear, high-level overview, explanation of how the feature or topic can help them on their journey, and context like use cases or examples. 
+- *Task* - Task content helps people complete a task from start to finish while they're using Aiven’s products.
+- *Reference* - Referential content provides detailed information that peoplr need while they're actively using Aiven's products.
+- *Tutorial* - Tutorials help people learn about products, and solve real world problems by guiding them through the entire workflow to complete a task.
+
+**What is the structure of devportal?**
+You can find the high-level content structure for the devportal in the following Slab article. 
+
+
+How can I contribute?
+---------------------
+
+**Reporting bugs, and suggesting enhancements**
+If you notice something is off (anything - from typos to incorrect instructions) or missing, consider opening an issue. 
+.. tip::
+    Before you open an issue, check the article request backlog, and all other open issues for the devportal. 
+
+**Contributing via pull requests (PR)**
+If you have your own article to add or you want to modify an existing article yourself, here's what you do:
+1. Fork the repo.
+2. Make your update.
+3. Open a pull request (make sure you use a helpful PR and commit message). 
+4. Submit your PR, and get it reviewed (one reviewer is required). 
+
+We'll merge it!
+
+.. note::
+    Our build runs several checks on your updates - from linting to spell check to broken links to help you commit good content. Look at the **Checks** section in your pull request to make sure everything passes. If anything fails, check the error details, or reach out to us, and we'll help you. 
+
+**Style tips**
+If you're writing content, here are some high-level tips that can be useful.
+
+- Decide what type of content you want to contribute (check content model), and use our content type templates - for concept, task, reference and tutorial.
```

Comment:
Is the content model something we'll include in this pull request? Or could we create a more bare bones `CONTRIBUTING` file to get us started and then revisit to add these resources when they're ready?

Reply:
I added the content model as a section in this file - **What is the content model?**. It's just here for the ease of use. I think we can keep it as it is now. And then update the file with links to templates (TBD).